cmd/execute.cpp

```diff
@@ -77,11 +77,14 @@ int main(int argc, char* argv[]) {
         return -1;
     }
 
-    // Check provided map size is valid
-    auto map_size{parse_size(map_size_str)};
-    if (!map_size.has_value()) {
-        SILKWORM_LOG(LogError) << "Invalid --lmdb.mapSize value provided : " << map_size_str << std::endl;
-        return -2;
+    std::optional<size_t> map_size{std::nullopt};
```

Comment:
The previous version was correct and shorter.
parse_size already returns 0 if string is empty

Reply:
Well, we want to leave the default mapSize value in lmdb::DatabaseConfig as is by default, not set it to 0, right? So the previous version wasn't quite correct when --lmdb.mapSize was not provided.
Also, I don't think that parse_size should return 0 if the input is empty. It should probably return std::nullopt since parse_size should only concern itself with parsing sizes, not setting defaults.